uber/site_sections/registration.py

```diff
@@ -24,11 +16,21 @@ def check_everything(attendee):
     if message:
         return message
 
-    if AT_THE_CON and not attendee.age_group and attendee.is_new:
+    if c.AT_THE_CON and not attendee.age_group and attendee.is_new:
         return "You must enter this attendee's age group"
 
 
-@all_renderable(PEOPLE, REG_AT_CON)
+def check_atd(func):
```

Comment:
Makes sense.  Now that we've unified the forms somewhat it probably won't matter as much, but I could definitely see that.  We can finalize something reasonable when we're closing to MAGStock, since that's when we'll really need to put the at-the-door stuff through the wringer and test it out more thoroughly.